themes-default/slim/views/addShows_newShow.mako

```diff
@@ -81,16 +97,6 @@ window.app = new Vue({
                 }
             });
 
-            setTimeout(() => {
-                this.updateBlackWhiteList();
-
-                const { providedInfo } = this;
-                const { use, showId, showDir } = providedInfo;
-                if (use && showId !== 0 && showDir) {
```

Comment:
Don't remove this part please, it is used to automatically go to the options section when you are adding a show from a folder, with provided metadata from `tvshow.nfo`